alephium/contracts/devnet/deploy_governance.ral

```diff
@@ -1,4 +1,4 @@
-TxScript Main(
+TxScript DevnetDeployGovernance(
```

Comment:
Why don't we use the same TxScripts to deploy to testnet and mainnet? Is it for testing purpose?

Since we now persist the deployment results, we might be able to remove these TxScripts.

Reply:
Because the contract ids will change every time we update the contract code, to avoid updating lots of configs, I introduce the devnet deployer, so that the contract ids will be the same except for the `DevnetDeployer` changes, I guess it's time to remove it because the contract code is close to finalized.